terraform/projects/infra-networking/main.tf

```diff
@@ -106,6 +108,7 @@ resource "aws_route53_zone" "subdomain" {
 resource "aws_route53_zone" "private" {
   vpc_id = "${module.vpc.vpc_id}"
   name   = "${local.private_subdomain_name}"
+  force_destroy = true
```

Comment:
Optional: I'm guessing this is added because we will have records added by the API call from the instances which otherwise would stop terraform a terraform destroy? Might be worth adding as a super brief inline comment or in the commit message just so it's clear why this line of code is here